### PR TITLE
chore: update repos

### PR DESCRIPTION
**Pull request by achrinza**
Update docs to reflect LoopBack 4 active development.

### CODE REVIEW

**Review by dhmlau — COMMENTED** [review]
@achrinza, thanks for the PR. I'm thinking whether it's better to update this page with LB4 info _and_ keep the LB3 content. According to https://strongloop.com/strongblog/lb3-entered-maintenance-mode/, we still accept PRs for LoopBack 3 till it reaches EOL in Dec 2020.

With this, I'd like to propose:
- mention LB4 is under active development, loopback-next, loopback-datasource-juggler, and most loopback-connector-* 
- LB3 is under maintenance LTS (critical bugs and security fixes only) but we still welcome contributions from the community. See details in https://strongloop.com/strongblog/lb3-entered-maintenance-mode/

What do you think?

**Comment by achrinza** [comment]
@dhmlau Good idea; I've applies your suggestions. WDYT?

File: pages/en/contrib/index.md [hunk 1]
```diff
@@ -8,13 +8,23 @@ permalink: /doc/en/contrib/index.html
...
-*   [loopback](https://github.com/strongloop/loopback)
-*   [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
-*   [loopback-boot](https://github.com/strongloop/loopback-boot)
-*   [loopback-phase](https://github.com/strongloop/loopback-phase)
-*   [generator-loopback](https://github.com/strongloop/generator-loopback)
+For LoopBack 4, components are stored in a monorepo. As such, all work on the project occurs in GitHub, primarily in the following repositories:
+
+- [loopback-next](https://github.com/strongloop/loopback-next)
+- [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
+- Most loopback-connector-*
```
Comment (dhmlau) [inline pages/en/contrib/index.md]: Perhaps like this? 
```
Most `loopback-connector-*` repos
```

**Review by dhmlau — APPROVED** [review]
@achrinza, your changes LGTM. I have a suggestion for one bullet. Feel free to incorporate. 

Also, while you're on this, do you mind update the LTS schedule on https://loopback.io/doc/en/contrib/Long-term-support.html where LB3 is supposed to be under `Maintenance LTS` instead of `Active LTS` now?  I'm good with having the current PR landed and have a follow up PR. Thanks.

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
@achrinza, thanks again for your contribution. Your PR has landed! 🎉

**Pull request merged**